test/assert.js

```diff
@@ -182,6 +183,29 @@ test('.throws()', function (t) {
 	t.end();
 });
 
+test('.throws() becomes the thrown Error', function (t) {
```

Comment:
Yeah, '.throws() returns the rejection reason of promise' is a correct version indeed.